Zend/zend.c

```diff
@@ -1650,6 +1650,20 @@ ZEND_API ZEND_COLD ZEND_NORETURN void zend_error_noreturn(int type, const char *
 	abort();
 }
 
+ZEND_API ZEND_COLD ZEND_NORETURN void zend_error_noreturn_unchecked(int type, const char *format, ...)
```

Comment:
what is the point of this function, it is 1:1 as `zend_error_noreturn` above

Reply:
The format attribute

Reply:
![image](https://github.com/php/php-src/assets/2228672/e0317f8f-bf7f-44c1-a292-81504111a3a6)